src/VS.Web.CG.Mvc/Templates/Identity/Bootstrap3/Pages/Account/Manage/Account.Manage.ExternalLogins.cs.cshtml

```diff
@@ -90,10 +90,10 @@ namespace @thisNamespace
         {
             // Clear the existing external cookie to ensure a clean login process
             await HttpContext.SignOutAsync(IdentityConstants.ExternalScheme);
-
+            var user = await _userManager.GetUserAsync(User);
             // Request a redirect to the external login provider to link a login for the current user
             var redirectUrl = Url.Page("./ExternalLogins", pageHandler: "LinkLoginCallback");
-            var properties = _signInManager.ConfigureExternalAuthenticationProperties(provider, redirectUrl, _userManager.GetUserId(User));
+            var properties = _signInManager.ConfigureExternalAuthenticationProperties(provider, redirectUrl, user.Id);
```

Comment:
Guess for this one, we should just use the one liner that already was here.  Sorry for the back and forth, I assumed we already had a user poco

Reply:
npnp